__tests__/comment/comment.mutations.test.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable no-undef */
+const { gql } = require('@apollo/client');
+const {
+  newComment,
+  updatedComment,
+  productId,
+  productWrongId,
+  commentWrongId,
+  rate,
+  updatedRate,
+} = require('./comment.variables');
+const {
+  COMMENT_NOT_FOUND,
+  COMMENT_FOR_NOT_EXISTING_PRODUCT,
+  RATE_FOR_NOT_EXISTING_PRODUCT,
+} = require('../../error-messages/comment.messages');
+const { setupApp } = require('../helper-functions');
+jest.mock('../../modules/upload/upload.service');
+
+let commentId = '';
+let operations;
+let productRate;
+let productRateCount;
+let productUserRates;
+describe('Comment queries', () => {
+  beforeAll(async () => {
+    operations = await setupApp();
+    const res = await operations
+      .query({
+        variables: { id: productId },
+        query: gql`
+          query($id: ID!) {
+            getProductById(id: $id) {
+              ... on Product {
+                rate
+                userRates {
+                  rate
+                }
+                rateCount
+              }
+              ... on Error {
+                message
+              }
+            }
+          }
+        `,
+      })
+      .catch(e => e);
+    const receivedProduct = res.data.getProductById;
+    productRate = receivedProduct.rate;
+    productRateCount = receivedProduct.rateCount;
+    productUserRates = receivedProduct.userRates;
+  });
+  it('1 should add a new comment', async () => {
```

Comment:
Please, remove `1` from the description string, check everywhere

Reply:
fixed